api4/team_test.go

```diff
@@ -1452,6 +1452,24 @@ func TestInviteUsersToTeam(t *testing.T) {
 			}
 		}
 	}
+
+	cfg := app.GetConfig()
+	cfg.TeamSettings.RestrictCreationToDomains = "@example.com"
+	cfg, _ = th.SystemAdminClient.UpdateConfig(cfg)
```

Comment:
This works but you can also directly set the config at `utils.Cfg` in tests, just a note for the future :)